tables/uk-comp.utb

```diff
@@ -46,6 +46,6 @@ nofor lowercase \x0457 1456		CYRILLIC SMALL LETTER YI
 nofor uppercase \x0490 124567		CYRILLIC CAPITAL LETTER GHE WITH UPTURN
 nofor lowercase \x0491 12456		CYRILLIC SMALL LETTER GHE WITH UPTURN
 
-include ru.ctb
-
 punctuation \x20b4 2578		HRYVNIA SIGN
+
+include ru.ctb
```

Comment:
No corresponding change in the test?

Reply:
What I meant to say was that there was already a test for the Hryvnia sign in the "forward only" section. Thought you might wanna change that into a "both directions" test.

Reply:
@bertfrees I don't think the test in forward section is excess. It's bit different from the test that I've added now.

Reply:
OK no problem.